pom.xml

```diff
@@ -32,15 +32,15 @@
     </developers>
 
     <properties>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>9</maven.compiler.release>
```

Comment:
We can achieve both Java 8 compatibility and Java 9+ modules using this: https://maven.apache.org/plugins/maven-compiler-plugin/examples/module-info.html